tests/tape_tests/test_ensemble.py

```diff
@@ -67,6 +67,64 @@ def test_from_parquet(data_fixture, request):
         assert parquet_ensemble._source[col] is not None
 
 
+def test_available_datasets(dask_client):
+    """
+    Test that the ensemble is able to successfully read in the list of available TAPE datasets
+    """
+    ens = Ensemble(client=dask_client)
+
+    datasets = ens.available_datasets()
```

Comment:
Best practices is usually to avoid external dependencies in unit tests because they increase run time and have the potential for flaky tests (if the external dependency is down). The alternative here would be to mock out the data set information and get call, which adds a lot of complexity. So I think we can leave this in there for now. But we should think if there is an alternative approach we can use for testing in the future.